src/server/audit/actions.ts

```diff
@@ -1,4 +1,14 @@
 export const AUDIT_ACTIONS = {
-  user: ["signed_in", "created"],
-  company: ["created"],
+  user: ["signup", "onboarded"],
+  company: ["create"],
+  stakeholder: [
+    "invite",
+    "re-invite",
+    "accept",
+    "update",
+    "remove",
+    "revoke-invite",
+    "deactivate",
+    "activate",
```

Comment:
Let's keep these events/actions consistent, lets go with `signed-up`, `invited`, `re-invited`, `accepted`. 

Note: these events has already occurred, it only makes sense to stick with past tense.